{% load static %}
<html>
	<head>
		<title>Dataset Details</title>
		<!--including meta tag required -->
		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <!--bootstrap link -->
	  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
		<!-- including a link to my css stylesheet -->
	  <link rel="stylesheet" href="{% static 'users/style.css' %}"/>
    <!-- allowing different styles for differently sized screens, like a laptop, mobile, desktop, tablets. -->
    <style>
      /*for mobile */
        @media only screen and (max-width:499px) {
            body{
                   font-style:Verdana;
                   background-color: #b0bf9e;
                }
        }
      /*for tablets */
        @media only screen and (min-width : 768px) and (max-width:1024px){
            body{
               	   font-style:Verdana;
                }
        }
      /*For Desktops and laptops */
        @media only screen and (min-width:1224px) {
            body{
                   background-color: #a2eef2;

                }
        }
        p::selection{
            	color:blue;
            	background:black;
        }
    </style>
  </head>
  <body>
  	<div class="dnabg"><br><br>
  	<h2 style="text-align:center;font-weight:bold;color:#c11f37;font-size:40px"><br><br> Details about Available Datasets </div>
  	<p>
    	We have hosted DNAse Footprint BED files that were produced as part of the ENCODE Project, contains deep sequencing DNase data that
    	will be used to identify sites where regulatory factors bind to the genome (footprints).
    	Footprinting is a technique used to define the DNA sequences that interact with and bind DNA-binding proteins,
    	such as transcription factors, zinc-finger proteins, hormone-receptor complexes, and other chromatin-modulating
    	factors like CTCF. The technique depends upon the strength and tight nature of protein-DNA interactions.
    	In their native chromatin state, DNA sequences that interact directly with DNA-binding proteins are
    	relatively protected from DNA-degrading endonucleases, while the exposed/unbound portions are readily degraded by such endonucleases.
    	A massively parallel next-generation sequencing technique to define the DNase hypersensitive sites in the
    	genome was adopted. The DNase samples were sequenced using next-generation sequencing machines to significantly higher depths of 300-fold or greater.
    	This produces a base-pair level resolution of the DNase susceptibility maps of the native chromatin state.
    	These base-pair resolution maps represent and are dependent upon the nature and the specificity of interaction of the DNA with the
    	regulatory/modulatory proteins binding at specific loci in the genome; thus they represent the native chromatin state of the genome under investigation. The deep sequencing approach has been
    	used to define the footprint landscape of the genome by identifying DNA motifs that interact with known or novel DNA-binding proteins.
  	</p>
  	<footer>
  		<footer id="basecolor">
  		<address style="color:white;text-align:center;font-weight:bold;font-size:24px;">
  		<!-- links to different pages on the website -->
  		<a href="{% url 'products' %}">Products</a> |
  		<a href="{% url 'knowledgecenter' %}">Knowledge Center</a> |
  		<a href="{% url 'contactus' %}">Contact Us</a>
			<a href="{% url 'logout' %}">Logout</a>
  	</footer>
	</body>
</html>
